app/controllers/base_trainee_controller.rb

```diff
@@ -227,6 +227,6 @@ def json_response
   end
 
   def render_partial(partial, locals)
-    (render_to_string(formats: %w[html], partial: partial, locals: locals) || "").squish
+    (render_to_string(formats: [:html], partial: partial, locals: locals) || "").squish
```

Comment:
@AbigailMcP This was the culprit preventing the filters from working. Rails 7 apparently only accepts symbols.